src/main/resources/db/changelog-backup/20260312_174006-dev-diff.yaml

```
@@ -0,0 +1,64 @@
+            type: datetime
+        tableName: admin_users
+- changeSet:
+    id: 1773304811015-1
+    author: gwanghyeon (generated)
+    changes:
+    - addNotNullConstraint:
+        columnDataType: boolean
+        columnName: is_public
+        tableName: event
+        validate: true
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)